package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "scripts": {
+    "build": "npm install && cd src/main/webapp && npm install && cd ../../..",
+    "dev": "concurrently 'cd src/main/webapp && npm run build:dev' 'mvn package appengine:run'",
+    "prod": "cd src/main/webapp && npm run build:prod && cd ../../../ && mvn package appengine:run",
+    "deploy": "cd src/main/webapp && npm run build:prod && cd ../../../ && mvn package war:war appengine:deploy"
+  },
+  "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^3.4.0",
+    "@typescript-eslint/parser": "^3.4.0",
+    "eslint": "^7.2.0",
+    "eslint-config-google": "^0.14.0",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-node": "^11.1.0",
+    "eslint-plugin-prettier": "^3.1.2",
+    "prettier": "^2.0.5",
+    "typescript": "^3.9.6"
+  },
+  "dependencies": {
```

Comment:
You're probably not using most of these dependencies, get rid of the ones you don't need.